drivers/watchdog/wdt_mcux_wwdt.c

```diff
@@ -86,6 +87,10 @@ static int mcux_wwdt_install_timeout(const struct device *dev,
 	clock_freq = CLOCK_GetWdtClkFreq(0);
 #elif defined(CONFIG_SOC_SERIES_RW6XX)
 	clock_freq = CLOCK_GetWdtClkFreq();
+#elif defined(CONFIG_SOC_SERIES_MCXNX4X)
+	const struct mcux_wwdt_config *config = dev->config;
+
+	clock_freq = CLOCK_GetWdtClkFreq(config->wdt_instance);
```

Comment:
feels like this section of the driver needs converted to use the clock control api